<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>883</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>深度学习软件</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,4,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,0,5">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>文件</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="styleSheet">
             <string notr="true">QTreeWidget#treeWidget{
	font: 16pt &quot;方正舒体&quot;;
	color: rgb(6, 64, 255);
	
	selection-color: rgb(255, 0, 4);
}

</string>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>数据集</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>图像</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="9,1">
          <item>
           <widget class="showGraph" name="image" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#image{
	background-color: rgba(255, 255, 255, 0);
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="9,1">
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>文件信息</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTreeWidget" name="treeWidget_2">
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6,3">
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="showGraph" name="graph" native="true">
        <property name="styleSheet">
         <string notr="true">showGraph#graph{
	background-color:rgba(255, 255, 255, 0)
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="previous">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="next">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="styleSheet">
         <string notr="true">QTextBrowser#textBrowser_2{
	font: 20pt &quot;方正舒体&quot;;
	color: rgb(255, 0, 0);
}</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="styleSheet">
         <string notr="true">QTableWidget#tableWidget_2{
	color: rgb(0, 26, 255);
	selection-color: rgb(255, 8, 0);
	font: 75 11pt &quot;Agency FB&quot;;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar#menubar {
    color: rgb(239, 238, 236);
    background-color: rgb(67, 61, 59);
}

QMenuBar#menubar::item {
    background-color: transparent;
}

QMenuBar#menubar::item:hover {
    color: rgb(255, 0, 0);
    background-color: transparent;
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="action_import_image"/>
     <addaction name="action_import_data"/>
     <addaction name="action_import_model"/>
     <addaction name="action_import_label"/>
     <addaction name="action_import_feature"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>导出</string>
     </property>
     <addaction name="action_result"/>
     <addaction name="action_export_labels"/>
     <addaction name="action_export_graph"/>
     <addaction name="action_export_features"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
    <addaction name="action_imageViewer"/>
    <addaction name="action_showFeature"/>
    <addaction name="action_dcm"/>
    <addaction name="action_cut"/>
    <addaction name="action_setup"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_information"/>
    <addaction name="action_help"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>影像组学流程</string>
    </property>
    <addaction name="action_import_dataset"/>
    <addaction name="action_extractFeature"/>
    <addaction name="separator"/>
    <addaction name="action_normalize"/>
    <addaction name="action_sift_statistic"/>
    <addaction name="action_sift_correlation_coefficient"/>
    <addaction name="action_sift_lasso"/>
    <addaction name="separator"/>
    <addaction name="action_predict"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_imageViewer"/>
   <addaction name="action_dcm"/>
   <addaction name="action_extractFeature"/>
   <addaction name="action_showFeature"/>
   <addaction name="action_cut"/>
   <addaction name="separator"/>
   <addaction name="action_setup"/>
   <addaction name="action_information"/>
  </widget>
  <action name="action_information">
   <property name="text">
    <string>版本号：2.0</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>预测</string>
   </property>
  </action>
  <action name="action_setup">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_imageViewer">
   <property name="text">
    <string>查看图像</string>
   </property>
  </action>
  <action name="action_dcm">
   <property name="text">
    <string>读取dcm文件</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_import_image">
   <property name="text">
    <string>图像</string>
   </property>
   <property name="toolTip">
    <string>导入图像</string>
   </property>
  </action>
  <action name="action_import_data">
   <property name="text">
    <string>数据集</string>
   </property>
   <property name="toolTip">
    <string>导入数据集</string>
   </property>
  </action>
  <action name="action_import_model">
   <property name="text">
    <string>模型</string>
   </property>
   <property name="toolTip">
    <string>导入模型</string>
   </property>
  </action>
  <action name="action_result">
   <property name="text">
    <string>结果</string>
   </property>
   <property name="toolTip">
    <string>导出结果</string>
   </property>
  </action>
  <action name="action_export_labels">
   <property name="text">
    <string>标签</string>
   </property>
   <property name="toolTip">
    <string>导出标签</string>
   </property>
  </action>
  <action name="action_export_graph">
   <property name="text">
    <string>图表</string>
   </property>
   <property name="toolTip">
    <string>图表</string>
   </property>
  </action>
  <action name="action_import_label">
   <property name="text">
    <string>标签</string>
   </property>
   <property name="toolTip">
    <string>导入标签</string>
   </property>
  </action>
  <action name="action_import_feature">
   <property name="text">
    <string>特征</string>
   </property>
   <property name="toolTip">
    <string>导入特征</string>
   </property>
  </action>
  <action name="action_showFeature">
   <property name="text">
    <string>查看特征</string>
   </property>
  </action>
  <action name="action_import_dataset">
   <property name="text">
    <string>导入数据集</string>
   </property>
  </action>
  <action name="action_extractFeature">
   <property name="text">
    <string>特征提取</string>
   </property>
  </action>
  <action name="action_normalize">
   <property name="text">
    <string>正则化</string>
   </property>
  </action>
  <action name="action_sift_statistic">
   <property name="text">
    <string>统计检验</string>
   </property>
  </action>
  <action name="action_sift_correlation_coefficient">
   <property name="text">
    <string>相关性系数-特征筛选</string>
   </property>
  </action>
  <action name="action_sift_lasso">
   <property name="text">
    <string>lasso回归-特征筛选</string>
   </property>
  </action>
  <action name="action_predict">
   <property name="text">
    <string>预测结果</string>
   </property>
  </action>
  <action name="actionsdf">
   <property name="text">
    <string>sdf</string>
   </property>
  </action>
  <action name="action_export_features">
   <property name="text">
    <string>特征</string>
   </property>
   <property name="toolTip">
    <string>导出特征</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>showGraph</class>
   <extends>QWidget</extends>
   <header>ui.ui.showGraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../texture/icons.qrc"/>
 </resources>
 <connections/>
</ui>
